pkg/framework/aries/default_srv.go

```diff
@@ -0,0 +1,18 @@
+// +build !js,!wasm
+
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package aries
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+	"github.com/hyperledger/aries-framework-go/pkg/storage/leveldb"
+)
+
+func storeProvider() (storage.Provider, error) {
```

Comment:
TODO:

- rename transient store to protocol state store (or similar).
- use indexedDB (as default) with a JS target.

(cc: @rolsonquadras )

Reply:
I add TODO for rename transient store to protocol state storehttps://github.com/hyperledger/aries-framework-go/pull/830/files#diff-7b2448fdc2181299533bc944239b9744R28

Reply:
use indexedDB (as default) with a JS target.  (DONE)